### PR TITLE
Doctor webapp

### PR DESCRIPTION
**Pull request by tejotemku**

### CODE REVIEW

**Review by noFrostoo — APPROVED** [review]

File: services/frontend/src/components/Navbar.vue [hunk 1]
```diff
@@ -1,40 +1,65 @@
...
+        <!-- future tabs
+        template 
+        <router-link to="    ">
+          <input 
+          class="tab"
+          type="button"
+          value="      "
+          />
+        </router-link>
+        -->
```
Comment (noFrostoo) [inline services/frontend/src/components/Navbar.vue:36]: please don't leave commented and unused code

File: services/frontend/src/components/SignIn.vue [hunk 2]
```diff
@@ -37,8 +41,37 @@
...
+  name: 'Login',
+  data() {
+    return {
+      username: null,
+      password: null
+    }
+  },
+  methods: {
+    checkLogin() {
+      // TODO: this is a mock
```
Comment (noFrostoo) [inline services/frontend/src/components/SignIn.vue:56]: Can you add a comment like this to other places with mocks ?

File: services/frontend/src/components/SignIn.vue [hunk 3]
```diff
@@ -37,8 +41,37 @@
...
+    },
+    acceptLogin() {
+      console.log('Logowanie powiodło się');
+      router.push('home')
+    },
+    rejectLogin() {
+      this.username=null;
+      this.password=null;
+      console.log('Logowanie nie powiodło się');
+      alert("Niepoprawny login");
```
Comment (noFrostoo) [inline services/frontend/src/components/SignIn.vue:72]: Do plan using alerts in the final version? Personally alerts just annoy me, i'd add notification system just to get rid of alerts, furthermore we  could use such system to give notifications then analysis has finished. Vue should have lib that will take care of that, and vuetify has nice notifications if we need them

**Review by noFrostoo — CHANGES_REQUESTED** [review]

File: services/frontend/src/components/SignIn.vue (hunk 2 above)
Comment (tejotemku) [reply services/frontend/src/components/SignIn.vue:56]: i think i did, but will check to be sure

File: services/frontend/src/components/SignIn.vue (hunk 3 above)
Comment (tejotemku) [reply services/frontend/src/components/SignIn.vue:72]: no i don't plan on that, but we need to show on monday that app will have some sort of feedback in that case, but got no time for something more elegant right now

File: services/frontend/src/components/SignIn.vue (hunk 2 above)
Comment (noFrostoo) [reply services/frontend/src/components/SignIn.vue:56]: ijn `verifyPatient` theres no comment

File: services/frontend/src/components/SignIn.vue (hunk 3 above)
Comment (noFrostoo) [reply services/frontend/src/components/SignIn.vue:72]: agree for now this will do

**Review by noFrostoo — APPROVED** [review]

**Pull request merged**